code/controllers/subsystem/garbage.dm

```diff
@@ -207,6 +210,14 @@ var/datum/subsystem/garbage_collector/SSgarbage
 		return TRUE
```

Comment:
I'm wondering if this should be 

`if(D.gc_destroyed && D.gc_destroyed != GC_CURRENTLY_BEING_QDELETED)`

because I'm not sure if `if(D.gc_destroyed)` will return true for a negative number

Basically, currently, calls to qdeleted(src) in Destroy return FALSE. I just want to make sure it stays that way so we don't run into hidden bugs later.